org.eclipse.xtext/src/org/eclipse/xtext/mwe/Reader.java

```diff
@@ -49,9 +49,20 @@
  * You might want to populate multiple workflow slots with model elements.
  * </p>
  * <p>
- * Below an example of a usage in the MWE XML syntax:
+ * Below are examples of a usage in the MWE2 DSL and MWE XML syntax:
  * </p>
  * <pre>
+ * component = org.eclipse.xtext.mwe.Reader {
+ *   useJavaClassPath = true
+ *   register = foo.bar.MyLanguageSetandaloneSetup {}
```

Comment:
Typo in StandaloneSetup

Reply:
copy/pasted the existing typo :/
fixed both now